src/include/sof/audio/drc/drc_plat_conf.h

```diff
@@ -0,0 +1,43 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Google LLC. All rights reserved.
+ *
+ * Author: Pin-chih Lin <johnylin@google.com>
+ */
+#ifndef __SOF_AUDIO_DRC_DRC_PLAT_CONF_H__
+#define __SOF_AUDIO_DRC_DRC_PLAT_CONF_H__
+
+/* Get platforms configuration */
+
+/* If next defines are set to 1 the DRC is configured automatically. Setting
+ * to zero temporarily is useful is for testing needs.
+ */
+#define DRC_AUTOARCH    1
+
+/* Force manually some code variant when DRC_AUTOARCH is set to zero. These
+ * are useful in code debugging.
+ */
+#if DRC_AUTOARCH == 0
+#define DRC_GENERIC	1
+#define DRC_HIFI3	0
+#endif
+
+/* Select optimized code variant when xt-xcc compiler is used */
+#if DRC_AUTOARCH == 1
+#if defined __XCC__
+#include <xtensa/config/core-isa.h>
+#if XCHAL_HAVE_HIFI3 == 1
```

Comment:
nit: in the future we should have this somewhere generic since this is a redefine of what is in IIR

Reply:
ack, this will probably have to come via Zephyr as there will likely be similar options in ARM